src/modules/welcome/modules/getStarted/controllers/GetStartedCtrl.js

```diff
@@ -0,0 +1,150 @@
+(function () {
+    'use strict';
+
+    const controller = function ($q, $mdDialog, apiWorker, $timeout, $state, user) {
+
+        const PATH = 'modules/welcome/modules/getStarted/templates';
+        const ORDER_LIST = [
+            'createAccount',
+            'backupEnter',
+            'backupWarning',
+            'backupSeed',
+            'backupSeedRepeat',
+            'backupSeedDone',
+            'end'
+        ];
+
+        class GetStartedCtrl {
+
+            constructor() {
+                this.stepIndex = 0;
+                this.password = '';
+                this.confirmPassword = '';
+                this.moneyInApp = false;
+                this.restoreByBackup = false;
+                this.hasAccount = false;
+                this.agree = false;
+                this.seed = '';
+                this.address = '';
+                this.seedList = [];
+
+                this.resetAddress();
+            }
+
+            setActiveSeed(item) {
+                const old = utils.find(this.seedList, { active: true });
+                if (old) {
+                    old.active = false;
+                }
+                item.active = true;
+                this.seed = item.seed;
+                this.address = item.address;
+            }
+
+            canConfirm() {
+                return !(this.moneyInApp && this.restoreByBackup && this.agree);
+            }
+
+            getStepUrl() {
+                return `${PATH}/${ORDER_LIST[this.stepIndex]}.html`;
+            }
+
+            clear() {
+                // TODO ???
+            }
+
+            next(index) {
+                this.checkNext().then(() => {
+                    if (index == null) {
+                        if (ORDER_LIST[this.stepIndex + 1]) {
+                            this.stepIndex++;
+                        } else {
+                            apiWorker.process((api, data) => {
+                                api.applySeed(data.seed);
+                                return api.encryptSeed(data.password);
+                            }, { seed: this.seed, password: this.password }).then((encryptSeed) => {
```

Comment:
Is it a method or a property? Maybe it should be `encryptedSeed`. Otherwise it reads as an action to be called.